WordPress/src/main/java/org/wordpress/android/ui/posts/PostUploadService.java

```diff
@@ -183,9 +183,14 @@ protected void onPostExecute(Boolean postUploadedSuccessfully) {
         @Override
         protected void onCancelled(Boolean aBoolean) {
             super.onCancelled(aBoolean);
-            mPostUploadNotifier.updateNotificationWithError(mErrorMessage, mIsMediaError, mPost.isPage(),
-                    mErrorUnavailableVideoPress);
-            WordPress.postUploadFailed(mPost.getLocalTableBlogId());
+            // mPostUploadNotifier and mPost can be null if onCancelled is called before doInBackground
+            if (mPostUploadNotifier != null) {
+                mPostUploadNotifier.updateNotificationWithError(mErrorMessage, mIsMediaError, mPost.isPage(),
```

Comment:
Is it possible for `mPostUploadNotifier` to be non-null when `mPost` is null? If so, calling `mPost.isPage()` here might result in an NPE.